package.json

```diff
@@ -10,7 +10,7 @@
     "extend": "^3.0.0",
     "google-auto-auth": "^0.5.2",
     "google-proto-files": "^0.9.1",
-    "grpc": "^1.1",
+    "grpc": "^1.2",
```

Comment:
I disagree with your conclusion; this _will_ force-upgrade anyone who runs `npm install` against the new version to 1.2.0, which we want (since 1.1.x had a bug).

I think we should accept the PR; I just want to make sure you are trying to pin `1.2.0 <= x < 2.0.0` and not `1.2.0 <= x < 1.3.0`. I think that is correct; I was just sanity checking.

(Unrelated: I disfavor `^` precisely because it is so often mis-read. In the future, I would support moving to explicit >= and < syntax.)